15.-Project-3-Elections-Scraper.py

```diff
@@ -0,0 +1,155 @@
+            print('Not correct number of arguments (2)')
+            break
+        else:
+            arg = sys.argv[1:]
+            count = 0
+            for k, v in payload.items():
+                payload[k] = arg[count]
+                count += 1
+            return payload
+
+
+def scrape_area():
+    parsed_command_line = parse_command_line()
+    r = requests.get(URL)
+    soup = BS(r.text, 'html.parser')
+    result_town = None
+    for table in soup.find_all('table'):
+        for child in filter(lambda x:x != '\n', table.children):
+            for info in filter(lambda x:x != '\n', child):
+                if info.string == parsed_command_line['area']:
+                    result_town = child
+
+    result_town_list = [i for i in filter(lambda x:x != '\n', result_town)]
+    area_link = result_town_list[-1].find('a').get('href')
+    area_url = "/".join(URL.split("/")[:-1]) + "/" + area_link
+    return area_url
+
+
+def scrape_municipality_url_id_name(scrape_area, parse_command_line):
+    parsed_command_line = parse_command_line
+    r = requests.get(scrape_area)
+    soup = BS(r.text, 'html.parser')
+    result_municipality = None
+    for table in soup.find_all('table'):
+        for child in filter(lambda x: x != '\n', table.children):
+            for info in filter(lambda x: x != '\n', child):
+                if info.string == parsed_command_line['municipality']:
+                    result_municipality = child
+
+    result_municipality_list = [i for i in filter(lambda x: x != '\n', result_municipality)]
+    municipality_id = result_municipality_list[0].string
+    municipality_name = result_municipality_list[1].string
+    municipality_link = result_municipality_list[0].find('a').get('href')
+    municipality_url = "/".join(URL.split("/")[:-1]) + "/" + municipality_link
+
+    return [municipality_url, municipality_id, municipality_name]
+
+
+def scrape_election_tables(scrape_municipality_url_id_name):
+    scrape_municipality = scrape_municipality_url_id_name
+    r =requests.get(scrape_municipality[0])
+    soup = BS(r.text, 'html.parser')
+    voters = []
+    tables = []
+    for table in soup.find_all('table'):
+        tables.append(table)
+
+    for row in filter(lambda x:x != '\n', tables[0].children):
+        try:
+            voters_in_the_list = row.find('td', {'class': 'cislo', 'headers':'sa2'}).string
+            voters_in_the_list = unicodedata.normalize("NFKD", str(voters_in_the_list))
+            distributed_voting_envelopes = row.find('td', {'class': 'cislo', 'headers':'sa3'}).string
+            distributed_voting_envelopes = unicodedata.normalize("NFKD", str(distributed_voting_envelopes))
+            valid_votes = row.find('td', {'class': 'cislo', 'headers':'sa6'}).string
+            valid_votes = unicodedata.normalize("NFKD", str(valid_votes))
+            voters.extend([voters_in_the_list, distributed_voting_envelopes,valid_votes])
+        except:
+            pass
```

Comment:
Don't do this. You won't know what is happening when your program won't behave and print no exception. Either you know what exceptions to ignore (and why) and then you should list them or you should log (print) the exception so you can notice.